content/administration/install/cdn.rst

```diff
@@ -1,80 +1,109 @@
-:guilabel:`CDN Base URL` field. This field is only visible and configurable if
-you have developer mode activated.
+In the Odoo :guilabel:`Website` app, go to the :menuselection:`Settings` and then activate the
+:guilabel:`Content Delivery Network (CDN)` setting and copy/paste the :guilabel:`Zone URL` value
+from the earlier step into the :guilabel:`CDN Base URL` field. This field is only visible and
+configurable when :doc:`../../applications/general/developer_mode` is activated.
 
-.. image:: cdn/odoo_cdn_base_url.png
-   :class: img-fluid
+.. note::
+   Ensure that there are two *forward slashes* (`//`) before the :guilabel:`CDN Base URL` and one
+   forward slash (`/`) after the :guilabel:`CDN Base URL`.
 
-Now your website is using the CDN for the resources matching the
-:guilabel:`CDN filters` regular expressions.
+:guilabel:`Save` the settings when complete.
 
-You can have a look to the HTML of your website in order to check if the CDN
-integration is properly working.
+.. image:: cdn/cdn-base-url.png
+   :align: center
+   :alt: Activate the CDN setting in Odoo.
 
-.. image:: cdn/odoo_check_your_html.png
-   :class: img-fluid
+Now the website is using the CDN for the resources matching the :guilabel:`CDN filters` regular
+expressions.
 
+In the HTML of the Odoo website, the :abbr:`CDN (content delivery network)` integration is evidenced
+as working properly by checking the :abbr:`URL (Uniform Resource Locators)` of images. The CDN Base
+URL can be seen by using your web browser's :guilabel:`Inspect` feature on the Odoo website. Look
+for it's record by searching within the :guilabel:`Network` tab inside of devtools.
 
-Why should I activate CORS?
----------------------------
+.. image:: cdn/test-pull.png
+   :align: center
+   :alt: The CDN Base URL can be seen using the inspect function on the Odoo website.
 
-A security restriction in some browsers (Firefox and Chrome at time of writing)
-prevents a remotely linked CSS file to fetch relative resources on this same
-external server.
+Prevent security issues by activating cross-origin resource sharing (CORS)
+--------------------------------------------------------------------------
 
-If you don't activate the CORS option in the CDN zone, the more obvious
-resulting problem on a default Odoo website will be the lack of font-awesome
-icons because the font file declared in the font-awesome CSS won't be loaded on
-the remote server.
+A security restriction in some browsers (such as Mozilla Firefox and Google Chrome) prevents a
+remotely linked CSS file to fetch relative resources on this same external server.
 
-Here's what you would see on your homepage in such a case:
+If the :abbr:`CORS (Cross-Origin Resource Sharing)` option isn't enabled in the :guilabel:`CDN
+Zone`, the more obvious resulting problem on a standard Odoo website will be the lack of Font
+Awesome icons because the font file declared in the Font Awesome CSS won't be loaded from the remote
+server.
```

Comment:
```suggestion
Zone`, the more obvious resulting problem on a standard Odoo website will be the lack of *Font
Awesome* icons because the font file declared in the *Font Awesome* CSS won't be loaded from the
remote server.
```